.github/workflows/codeql-analysis.yml

```diff
@@ -37,15 +37,15 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5
```

Comment:
How did you obtain these hashes?

Reply:
https://github.com/actions/checkout/releases/tag/v2.7.0 and then click the small hash to get https://github.com/actions/checkout/commit/ee0669bd1cc54295c223e0bb666b733df41de1c5 , then copy the long sha